DESCRIPTION

```diff
@@ -40,6 +40,7 @@ Suggests:
     desc,
     testthat
 Remotes:
+  joeHickson/futile.logger@i67_ftry,
```

Comment:
I just merged the branch into the master to resolve this but that also works 👍

Reply:
Nice - updated to use your master. Hopefully he will respond to the PR soon.